src/components/ec-input-field/ec-input-field.vue

```diff
@@ -135,6 +139,8 @@ const props = withDefaults(defineProps<InputFieldProps>(), {
   isLoading: false,
   isSensitive: false,
   isWarning: false,
+  bgGrayColorLevel: 8,
+  showPointerCursor: false,
```

Comment:
why do we need this prop?

Reply:
we don't want to show the pointer cursor for a normal input, but we do show it if it's a dropdown input

Reply:
I think we should always show it if the content doesn't fit.